tests/core/test_pod_security_policies.py

```diff
@@ -141,3 +143,30 @@ def pspt_del_check():
     # will timeout if pspt is not deleted
     wait_for(pspt_del_check)
     assert api_client.by_id_pod_security_policy_template(pspt_proj.id) is None
+
+
+def test_incorrect_pspt(admin_mc, remove_resource):
+    """ Test that incorrect pod security policy templates cannot be created"""
+    api_client = admin_mc.client
+
+    name = "pspt" + random_str()
+    with pytest.raises(ApiError) as e:
+        api_client.create_podSecurityPolicyTemplate(name=name)
+    assert e.value.error.status == 422
+
+    name = "pspt" + random_str()
+    with pytest.raises(ApiError) as e:
+        runas = {"rule": "RunAsAny"}
+        selinx = {"rule": "RunAsAny"}
+        supgrp = {"rule": "RunAsAny"}
+        fsgrp = {"rule": "MustRunAs"}
+        # Should not set the default True if allowPrivilegedEscalation is false
+        api_client.\
+            create_podSecurityPolicyTemplate(name=name, fsGroup=fsgrp,
+                                             runAsUser=runas,
+                                             seLinux=selinx,
+                                             supplementalGroups=supgrp,
+                                             privileged=False,
+                                             defaultAllowPrivilegeEscalation=True)  # noqa: E501
```

Comment:
The line is slightly too long here but this is more readable than using a variable.

Reply:
I don't agree with doing this, create a dict with all your info and pass that in instead. Any changes to this is going to cause the next person issues as well.

Reply:
Amended to use dict